code/game/objects/items/mop.dm

```diff
@@ -14,16 +14,26 @@
 	attack_verb_simple = list("mop", "bash", "bludgeon", "whack")
 	resistance_flags = FLAMMABLE
 	var/mopcount = 0
+<<<<<<< HEAD
 	var/mopcap = 50 //SKYRAT EDIT CHANGE
 	var/mopspeed = 15
+=======
+	///Maximum volume of reagents it can hold.
+	var/max_reagent_volume = 15
+	var/mopspeed = 1.5 SECONDS
+>>>>>>> a3a40e2994f (Gives Omega soap 800 uses instead of 301 (#64813))
```

Comment:
```suggestion
	///Maximum volume of reagents it can hold.
	var/max_reagent_volume = 50 // SKYRAT EDIT - ORIGINAL: 15
	var/mopspeed = 1.5 SECONDS
```